field/src/field.rs

```diff
@@ -288,57 +455,67 @@ pub trait Field:
 {
     type Packing: PackedField<Scalar = Self>;
 
-    /// A generator of this field's entire multiplicative group.
+    /// A generator of this field's multiplicative group.
     const GENERATOR: Self;
 
+    /// Check if the given field element is equal to the unique additive identity (ZERO).
     fn is_zero(&self) -> bool {
         *self == Self::ZERO
     }
 
+    /// Check if the given field element is equal to the unique multiplicative identity (ONE).
     fn is_one(&self) -> bool {
         *self == Self::ONE
     }
 
```

Comment:
These also make sense for rings that are not fields. Should they be part of the ring trait?